src/views/PopularMovieView.vue

```diff
@@ -2,17 +2,20 @@
   <div>
     <div class="title text-center text-h4 font-weight-bold green--text text--darken-3">{{titleForMoviePage}}</div>
     <MoviesList :movies="popularMovies" />
```

Comment:
Please rename this component to `PopularMoviesView`

Reply:
Done